src/renderer/pages/Staking/Overview/components/NominatorsList/NominatorsList.tsx

```diff
@@ -109,77 +90,29 @@ export const NominatorsList = ({
-                <HelpText className="text-text-positive">{t('staking.tooltips.redeemTitle')}</HelpText>
-              </div>
-            </Tooltip>
-          );
-
-          const content = (
-            <>
-              <AccountAddress className="max-w-[115px]" name={stake.accountName} address={stake.address} />
-              <div className="ml-auto">{unstakeBadge || redeemBadge}</div>
-            </>
-          );
-
-          return (
-            <li key={stake.id}>
-              <Plate className="grid grid-cols-[226px,104px,104px,40px] items-center gap-x-6">
-                {!walletUtils.isWatchOnly(activeWallet) && nominators.length > 1 ? (
-                  <Checkbox
-                    disabled={isStakingLoading}
-                    checked={stake.isSelected}
-                    onChange={() => onToggleNominator(stake.address)}
-                  >
-                    {content}
-                  </Checkbox>
-                ) : (
-                  <div className="flex items-center gap-x-2">{content}</div>
-                )}
-                <div className="justify-self-end flex flex-col items-end">
-                  {!stake.totalStake || !asset ? (
-                    <>
-                      <Shimmering width={82} height={20} />
-                      <Shimmering width={56} height={18} />
-                    </>
-                  ) : (
-                    <>
-                      <AssetBalance value={stake.totalStake} asset={asset} />
-                      <AssetFiatBalance amount={stake.totalStake} asset={asset} />
-                    </>
-                  )}
-                </div>
-                <div className="justify-self-end flex flex-col items-end">
-                  {!stake.totalReward || !asset ? (
-                    <>
-                      <Shimmering width={82} height={20} />
-                      <Shimmering width={56} height={18} />
-                    </>
-                  ) : (
-                    <>
-                      <AssetBalance value={stake.totalReward} asset={asset} />
-                      <AssetFiatBalance amount={stake.totalReward} asset={asset} />
-                    </>
-                  )}
-                </div>
-                <InfoPopover data={getExplorers(stake.address, stake.stash, explorers)} position="top-full right-0">
-                  <Icon name="info" size={14} />
-                </InfoPopover>
-              </Plate>
+          return Array.isArray(stake) ? (
+            <ShardedList
+              isStakingLoading={isStakingLoading}
+              shardsStake={stake}
+              era={era}
+              asset={asset}
+              explorers={explorers}
+              getContent={getContent}
+              onToggleNominator={onToggleNominator}
+              onCheckValidators={onCheckValidators}
+            />
+          ) : (
+            <li key={(stake.account as Account).id} className={cnTw(isShards && '[&>*:first-child]:pl-9')}>
```

Comment:
`[&>*:first-child]:pl-9` what you were trying to achieve? We can try to simplify this 😉

Reply:
I don't want to send down an additional prop but if we have shards on the screen all other accounts should have additional padding on the left